battleship.py

```diff
@@ -62,13 +62,13 @@ def mousePressed(data, event, board):
 Returns: 2D list of ints
 '''
 def emptyGrid(rows, cols):
-    list_1=[]
+    grid=[]

```

Comment:
Thank you @priyusha99 
You can merge the code to the main branch.

Reply:
okay sir thank you @LaxmiNarayanaMurthyVemuri